acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -135,16 +136,10 @@ func TestMeshGatewayDefault(t *testing.T) {
 // with ACLs and TLS with and without auto-encrypt enabled.
 func TestMeshGatewaySecure(t *testing.T) {
 	cases := []struct {
-		name              string
-		enableAutoEncrypt string
+		name string
 	}{
 		{
-			"with ACLs and TLS without auto-encrypt",
-			"false",
-		},
-		{
-			"with ACLs and auto-encrypt",
-			"true",
+			"with ACLs and TLS",
```

Comment:
not a blocker, but it'd be nice to join these two tests together

Reply:
done!